language/move-lang/src/to_bytecode/translate.rs

```diff
@@ -733,12 +733,12 @@ fn fun_type_parameters(tps: Vec<TParam>) -> Vec<(IR::TypeVar, BTreeSet<IR::Abili
         .collect()
 }
 
-fn struct_type_parameters(tps: Vec<TParam>) -> Vec<IR::StructTypeParameter> {
+fn struct_type_parameters(tps: Vec<StructTypeParameter>) -> Vec<IR::StructTypeParameter> {
     tps.into_iter()
         .map(|tp| {
-            let name = type_var(tp.user_specified_name);
-            let constraints = abilities(&tp.abilities);
-            (/* is_phantom */ false, name, constraints)
+            let name = type_var(tp.param.user_specified_name);
```

Comment:
not as big of a deal as ast_debug case. But a pattern match might be nice here too